rootfs/etc/apache2/sites-enabled/000-default.conf

```diff
@@ -0,0 +1,29 @@
+<VirtualHost *:80>
```

Comment:
Do we need anything else like it's already enabled on our owncloud base image? https://github.com/owncloud-docker/base/blob/master/rootfs/etc/apache2/sites-enabled/000-default.conf

Reply:
This header enforces a https connection